wp-content/themes/bh/views/donate/thank-you.php

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
Has this page defined outside the theme template, on purpose?
Is there a way to go back to the website after a donation has placed?